app/controllers/dashboard/profile_controller.rb

```diff
@@ -13,10 +13,31 @@ def update
     end
   end
 
+  def invite_friends
+    @incorrect_emails = FriendsService.invite(invite_friends_params[:emails])
+    if @incorrect_emails.blank?
+      flash.now[:notice] = 'Ваши друзья успешно приглашены на сайт.'
+    else
+      flash.now[:alert] = 'Проверьте формат вводимых данных: ' + @incorrect_emails + '.'
+      flash.now[:notice] = nil
+    end
+
+    respond_to do |format|
+      format.html
+      format.js
+    end
+  end
+
   private
 
   def user_params
     params.require(:user).permit(:email, :password, :password_confirmation,
                                  :locale)
   end
+
+  def invite_friends_params
+    parameters = params.require(:invite_friends).permit(:emails)
+    parameters[:emails] = parameters[:emails].gsub(' ', '').split(',')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.